src/utils.py

```diff
@@ -39,17 +38,32 @@ def format_token_short(token):
 def format_integer(number):
   return '{:,d}'.format(number)
 
-def format_amount_in_weis(amount, decimals, rounding=ROUND_DOWN):
-  value = amount / Decimal(10 ** decimals)
-  return format_amount(value, decimals=decimals, rounding=rounding)
+def isUnlimitedAmount(amount):
+  global MAX_AMOUNT
+
+  return amount == MAX_AMOUNT
+
+def format_amount_in_weis(amount, decimals, rounding=ROUND_DOWN, unlimitedLabel='Unlimited'):
+  if isUnlimitedAmount(amount):
+    return unlimitedLabel
+  else:
+    value = amount / Decimal(10 ** decimals)
+    return format_amount(value, decimals=decimals, rounding=rounding)
 
 def format_price(amount, decimals=10, rounding=ROUND_DOWN, currency=''):
   price = format_amount(amount, decimals=decimals, rounding=rounding)
   return price + ' ' + currency if currency else price
 
+def format_percentage(value, total):
+  if isUnlimitedAmount(total):
+    return ''
+  else:
+    percentage = (Decimal(value) / Decimal(total)) * Decimal(100)
+    return format_amount(percentage, decimals=2) + '%'
```

Comment:
Guess what? You can use f-strings for formatting the percentage for you. Examples https://stackoverflow.com/a/59591771/1272513

Reply:
https://github.com/gnosis/dex-cli/issues/39